tests/unit/test_utils_distutils_args.py

```diff
@@ -60,4 +60,4 @@ def test_all_value_options_work(name: str, value: str) -> None:
 
 def test_user_option_works() -> None:
     result = parse_distutils_args(["--user"])
-    assert result["user"] == 1
+    assert result["user"] == "true"
```

Comment:
So we don’t care about this value after the option is parsed?

Reply:
@uranusjr it seems that the call sites only care about the presence of the options, and not their values, so this change should not matter in practice.

Reply:
In that case perhaps it’s better to change the test to `assert "user" in result` to reflect that?

Reply:
I made this `"true"` to avoid needing to add `Literal[1]` to the `Dict[str, str]` return type of this function.

Reply:
I've updated the assertion to `assert result["user"]` to reflect that we only care about it existing and being truthy.